thunder/__init__.py

```diff
@@ -155,17 +156,22 @@ def __version__():
 get_default_executors = extend.get_default_executors
 get_always_executors = extend.get_always_executors
 
+cudnn_executor: None | extend.Executor = extend.get_executor("cudnn")
 sdpa_executor: None | extend.Executor = extend.get_executor("sdpa")
 nvfuser_executor: None | extend.Executor = extend.get_executor("nvfuser")
 pytorch_executor: None | extend.Executor = extend.get_executor("torch")
 
-# Default executor list is [sdpa -> nvfuser -> torch -> python]
+# Default executor list is [cudnn -> sdpa -> nvfuser -> torch -> python]
```

Comment:
Why do we still run sdpa by default in addition to cudnn? Isn't that redundant?

Reply:
I think the reason is that cuDNN's coverage of shapes/dtypes is smaller than PyTorch's SDPA and we'd like to use it when it happens. Then if PyTorch's SDPA can't run the operation we'd decompose it and use nvFuser+PyTorch for matmul.

Reply:
Yes, I did not want to suddenly cause slowness for shapes not supported by cudnn's sdpa.